woocommerce/woocommerce-gutenberg-products-block
# Add dropdown display style to Attribute Filter block

Part of #1134 and #1135.

This PR implements the dropdown display style for the _Filter Products by Attribute_ block with the _Query Type_ set to `OR` (see #1134 for the difference between `OR` and `AND`).

I plan to implement the `AND` format in a follow-up, but wanted to get feedback early before starting with it.

#### Accessibility

### Screenshots
![Peek 2019-11-25 20-03](https://user-images.githubusercontent.com/3616980/69569869-db8ee780-0fbe-11ea-80f9-52fd95c9be20.gif)

### How to test the changes in this Pull Request:

1. Create a post with a _Filter Products by Attribute_ block and select the _Dropdown_ option in _Display Style_.
2. Preview the post and interact with the filter (search terms, add them, remove them, repeat only using the keyboard, using a screen reader, etc.).
3. Verify everything works as expected.

### Follow-ups
* As mentioned, the `AND` dropdown format will be implemented in a follow-up.
* I didn't implement the _Apply_ button that can be seen in the mockups. I would like to discuss with @jwold not showing it or making it optional as in the _Price Filter_, so instant updates can happen.

### Changelog

> Add dropdown display style to Filter Products by Attribute block.
